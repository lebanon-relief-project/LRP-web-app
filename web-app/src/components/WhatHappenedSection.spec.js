import React from "react";
import { render } from "@testing-library/react";
import WhatHappened from "./WhatHappenedSection";
import "jest-styled-components";

describe("the WhatHappened component", () => {
  let container;
  it("should match the snapshot", () => {
    ({ container } = renderWhatHappenedSection());

    expect(container).toMatchSnapshot();
  });
});

const renderWhatHappenedSection = () => {
  return render(<WhatHappened />);
};
