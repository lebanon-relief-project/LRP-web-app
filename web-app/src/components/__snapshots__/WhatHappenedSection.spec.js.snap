// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`the WhatHappened component should match the snapshot 1`] = `
.c4 {
  margin-top: 10rem;
  max-width: 30rem;
}

.c3 {
  padding-left: 10rem;
}

.c2 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
}

.c1 {
  background: inherit;
  padding-left: 10rem;
}

.c0 {
  margin: 0;
  padding: 0;
}

.c5 {
  font-family: Playfair Display;
  font-style: normal;
  font-weight: 900;
  font-size: 38px;
  line-height: 46px;
  color: #000000;
}

.c6 {
  margin: 0;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  font-family: Raleway;
  font-style: normal;
  font-weight: 600;
  font-size: 16px;
  color: #000000;
}

@media (max-width:576px) {
  .c4 {
    margin-top: 0rem;
    max-width: 100%;
  }
}

@media (max-width:576px) {
  .c3 {
    max-width: 100%;
    padding-left: 0rem;
  }
}

@media (max-width:576px) {
  .c2 {
    -webkit-flex-direction: column;
    -ms-flex-direction: column;
    flex-direction: column;
  }
}

@media (max-width:576px) {
  .c1 {
    padding-left: 0rem;
  }
}

<div>
  <section
    class="c0"
  >
    <div
      class="c1"
    >
      <div
        class="c2"
      >
        <div>
          <svg
            class="c3"
          >
            Illustration1.svg
          </svg>
        </div>
        <div
          class="c4"
        >
          <h2
            class="c5"
          >
            What happened
          </h2>
          <p
            class="c6"
          >
            On August 4 an explosion shook Beirut. It was the largest non-nuclear explosion ever recorded. That day 200 people died, 6,500 were injured, and 300,000 were left homeless. This project was developed to support those affected and provide a platform for the community to come together and help each other heal.
          </p>
        </div>
      </div>
    </div>
  </section>
</div>
`;
